youtube_dl/extractor/nhk.py

```diff
@@ -0,0 +1,29 @@
+from __future__ import unicode_literals
+
+from .common import InfoExtractor
+
+
+class NhkVodIE(InfoExtractor):
+    _VALID_URL = r'http://www3.nhk.or.jp/nhkworld/en/vod/(?P<id>.+)\.html'
+    _TESTS = [{
+        'url': 'http://www3.nhk.or.jp/nhkworld/en/vod/tokyofashion/20160815.html',
+        'info_dict': {
+            'id': 'A1bnNiNTE6nY3jLllS-BIISfcC_PpvF5',
+            'ext': 'flv',
+            'title': '[nhkworld]VOD;2009-251-2016;TOKYO FASHION EXPRESS;The Kimono as Global Fashion;en',
+        },
+        'params': {
+            'skip_download': True  # Videos available only for a limited period of time.
+        },
+    }]
+
+    def _real_extract(self, url):
+        video_id = self._match_id(url)
+        webpage = self._download_webpage(url, video_id)
+
+        embed_code = self._search_regex(
+            r'''nw_vod_ooplayer\('movie-area', '([^']+)'\);''',
```

Comment:
The original version had double quotes, which was commented on by @TRox1972 (I received the notification email from Github). I think he removed the comment later, though.

After all I think I prefer the triple single quotes over double quotes here anyway, as they more ambiguously indicate that the string contains special characters (be it line break or normal delimiter).

Would you consider it acceptable or do I have to change it back to double quotes or single quotes with every inner single quote escaped (later being IMHO more disruptive to read)?